server/monitor/src/main/java/org/apache/accumulo/monitor/Monitor.java

```diff
@@ -647,6 +648,7 @@ private void getMonitorLock() throws KeeperException, InterruptedException {
       }
 
       if (!monitorLockWatcher.failedToAcquireLock) {
+        log.warn("Failed to acquire Monitor lock. Monitor lock is in unknown state.");
```

Comment:
IMO, this isn't adding any value. The thrown unchecked exception is already propagating what the log message says. As an operator, if I saw that exception logged, the log message wouldn't tell me anything else about what went wrong or how to fix it.